src/API/api.ts

```diff
@@ -0,0 +1,8 @@
+const API_URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=9d3a21ab';
```

Comment:
You are using default link which was send to you after registration. default link already get you on the one and only movie  Try to find out by yourself how to use the api-link correctly. Couse in link the part `i=tt389619` means id of movie and `key=9d3a21ab` was send to you separately in the same massage as the link.